elastio-terraform-deployment/module/main.tf

```diff
@@ -0,0 +1,95 @@
+locals {
+  elastio_endpoint = "https://${var.elastio_tenant}/public-api/v1"
+  headers = {
+    Authorization = "Bearer ${var.elastio_pat}"
+  }
+}
+
+data "http" "cloudformation_template" {
+  url             = "${local.elastio_endpoint}/cloudformation-template"
+  request_headers = local.headers
+
+  retry {
+    attempts     = 10
+    max_delay_ms = 10000
+  }
+
+  lifecycle {
+    postcondition {
+      condition     = self.status_code >= 200 && self.status_code < 300
+      error_message = "Failed to fetch CloudFormation template"
+    }
+  }
+}
+
+resource "aws_cloudformation_stack" "elastio_account_level_stack" {
+  name         = "elastio-account-level-stack"
+  template_url = data.http.cloudformation_template.response_body
+  tags = {
+    "elastio:resource" = "true"
+  }
+  capabilities = ["CAPABILITY_NAMED_IAM"]
+}
+
+resource "aws_cloudformation_stack" "elastio_nat_provision_stack" {
+  count = var.elastio_nat_provision_stack == null ? 0 : 1
+
+  name = "elastio-nat-provision-lambda"
+  template_url = join(
+    "/",
+    [
+      "https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com",
```

Comment:
There shouldn't be any issue w/ deployments to some other region using the `us-east-2` bucket, right?

Reply:
Yes, the Cloudformation template URL can be in any region